packages/mds-geography-service/@types/index.ts

```diff
@@ -61,6 +61,7 @@ export interface GeographyService {
   getPublishedGeographies: (options?: GetPublishedGeographiesOptions) => GeographyWithMetadataDomainModel[]
   writeGeographies: (geographies: GeographyDomainCreateModel[]) => GeographyDomainModel[]
   writeGeographiesMetadata: (metadata: GeographyMetadataDomainCreateModel[]) => GeographyMetadataDomainModel[]
+  editGeography: (geography: GeographyDomainCreateModel) => GeographyDomainModel
```

Comment:
why is it `CreateModel` for `edit`? Shouldn't the incrementing ID and such be set at that point?

Reply:
it uses a UUID externally provided, rather than an incrementing ID. :)